runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc

```diff
@@ -2683,5 +2683,136 @@
       "z1": 0,
       "z2": 0
     }
+  ],
+  "10835": [ // Dorgesh-Kaan North
+    {
+      "rx1": 14,
+      "ry1": 16,
+      "rx2": 14,
+      "ry2": 24,
+      "z1": 1,
+      "z2": 1
+    },
```

Comment:
Yet another spot it would be nice down the line to be able to negate the map's given roof flags.